lib/inets/src/http_client/httpc.erl

```diff
@@ -765,12 +765,14 @@ http_options_default() ->
 		error
 	end,
 
+    Ssl = httpc:ssl_verify_host_options(true),
+
```

Comment:
Why is this an external call? It could be` Ssl = ssl_verify_host_options(true),` ?!